services/github/github-gist-last-commit.service.js

```diff
@@ -0,0 +1,47 @@
+import Joi from 'joi'
+import { formatDate } from '../text-formatters.js'
+import { age as ageColor } from '../color-formatters.js'
+import { GithubAuthV3Service } from './github-auth-service.js'
+import { documentation, errorMessagesFor } from './github-helpers.js'
+
+const schema = Joi.object({
+  updated_at: Joi.string().required(),
+}).required()
+
+export default class GithubGistLastCommit extends GithubAuthV3Service {
+  static category = 'activity'
+  static route = { base: 'github/gists', pattern: ':gistId' }
```

Comment:
We'll need to adjust the route here though I'm not entirely sure to what just yet.

We've already got a few more requests for gist-related badges, and our [docs on badge urls](https://github.com/badges/shields/blob/master/doc/badge-urls.md) prescribe a pattern we'll need to follow.

I suspect we'll probably want to do something like:

```suggestion
  static route = { base: 'github-gist/last-commit', pattern: ':gistId' }
```

Though I suppose we _could_ also consider doing a secondary noun, e.g.

```suggestion
  static route = { base: 'github/gist/last-commit', pattern: ':gistId' }
```

I'd suggest the former for now, but we can see what the other maintainers think since whatever we choose here will likely end up setting the standard for gist-related badges

Reply:
Cool, I applied the first suggestion.
If you and the other maintainers decide to go with the second option, I will update it accordingly :)